### ISSUE TITLE
Total frames no longer editable in stack import tool

### ISSUE THREAD

**Issue by joelvbernier** [HEXRD/hexrdgui#918]
Trying the previous test with 208 tiff files for one detector; suddenly the spin controller for total frames is initialized to 1 and not editable, preventing import.
![image](https://user-images.githubusercontent.com/1154130/117083134-d2985e80-acf8-11eb-9d3e-c6474dbf37c3.png)

**Comment by joelvbernier** [comment]
Update:  So if I change the "max file frames" to 1 (0 is supposed to mean "all"), then it autopopulates "Total Frames".  Actually, "Total Frames" is a non-interactive counter, correct?  Should we just put a text box there?

**Comment by joelvbernier** [comment]
More badness here.  I loaded an example with 208 tiff files (1 frame per file).  Then I try to load a case with the 2 GE files, where each file has
- 242 total frames
- 2 empty at the start
![image](https://user-images.githubusercontent.com/1154130/117172050-4af63280-ad80-11eb-9db9-63b0a7d1e51e.png)
yet when I try to load, I am getting an `OmegaImageseriesError` seemingly because the data structure behind the widget is remembering the previous values instead of taking the ones from the displayed interactions.
```
Traceback (most recent call last):
  File "c:\users\berni\documents\github\hexrdgui\hexrd\ui\async_worker.py", line 73, in run
    result = self.fn(*self.args, **self.kwargs)
  File "c:\users\berni\documents\github\hexrdgui\hexrd\ui\image_load_manager.py", line 194, in process_ims
    self.add_omega_metadata(HexrdConfig().imageseries_dict)
  File "c:\users\berni\documents\github\hexrdgui\hexrd\ui\image_load_manager.py", line 337, in add_omega_metadata
    ims_dict[key].metadata['omega'] = omw.omegas
  File "c:\users\berni\documents\github\hexrd\hexrd\imageseries\omega.py", line 145, in omegas
    raise OmegaSeriesError(msg)
hexrd.imageseries.omega.OmegaSeriesError: 'number of frames (208) does not match number of wedge frames (480)'
```
if I close it and open it again -- changing nothing -- then it seems to finally sync.  So something is clearly not collecting the signals from the interactions properly.

**Comment by joelvbernier** [comment]
And to reiterate, "Total Frames" is not updating upon new file selection (this is just reporting the total number of frames in the selected files, right?)

**Comment by bnmajor** [comment]
Closing, this was fixed in #921
